internal/testutils/xds/e2e/clientresources.go

```diff
@@ -700,6 +700,10 @@ type BackendOptions struct {
 	HealthStatus v3corepb.HealthStatus
 	// Weight sets the backend weight. Defaults to 1.
 	Weight uint32
+	// Additional port number on which the backend is accepting connections.
+	// All backends are expected to run on localhost, hence host name is not
+	// stored here.
+	AdditionalPorts []uint32
```

Comment:
I'm ok with having this here since if we change the `Port` field to be `Ports` instead and make it a slice, that would mean changing a *lot* of tests. But we can discuss about the change to the internal eds resource struct in the other comment.

Reply:
Changed the existing `Port` fields to become a slice. It didn't take too much time after using vim macros after visiting every reference from the LSP.